ci-operator/step-registry/ipi/install/loki/ipi-install-loki-commands.sh

```diff
@@ -589,8 +576,14 @@ spec:
         app.kubernetes.io/version: ${LOKI_VERSION}
     spec:
       containers:
-      - args:
-        - "-config.file=/etc/promtail/promtail.yaml"
+      - command:
+        - sh
+        - -c
+        - |
+          promtail \
+            -client.url=http://loki.loki.svc.cluster.local:3100/api/prom/push \
+            -client.external-labels=_id=\$(cat /tmp/shared/cluster-id),host=\$(HOSTNAME) \
```

Comment:
Do you really want to use `_id` als label name here? IIRC prometheus metrics, e.g. cadvisor use `cluster` for such a case. Thinking loudly here, it might be good to align such labels with the metrics of the same CI cluster.

Reply:
We're going to pair it with Telemetry results, which have `_id` so I guess that would be easier to use the same label